pkg/envoy/lds/ingress_test.go

```diff
@@ -163,23 +149,12 @@ func TestGetIngressFilterChainFromTrafficMatch(t *testing.T) {
 	for i, tc := range testCases {
 		t.Run(fmt.Sprintf("Testing test case %d: %s", i, tc.name), func(t *testing.T) {
 			assert := tassert.New(t)
-			mockCtrl := gomock.NewController(t)
-			defer mockCtrl.Finish()
-
-			mockConfigurator := configurator.NewMockConfigurator(mockCtrl)
 
 			lb := &listenerBuilder{
-				serviceIdentity: tests.BookstoreServiceIdentity,
-				cfg:             mockConfigurator,
+				proxyIdentity: tests.BookstoreServiceIdentity,
 			}
 
-			mockConfigurator.EXPECT().IsTracingEnabled().Return(false)
```

Comment:
dropping this seems like a nice improvement, making the tests easier to debug/comprehend 💯